tests/integration/conftest.py

```diff
@@ -49,6 +49,11 @@ def real_dynamo() -> bool:
     return os.environ.get("TEST_ON_AWS", "false") == "true"
 
 
+@pytest.fixture(scope="session")
+def is_dynalite_emulator() -> bool:
+    return os.environ.get("DYNALITE_EMULATOR", "false") == "true"
```

Comment:
I think it would be better to do feature detection rather than relying on the person/system running the system setting an env var. Maybe something like this:

```python
@pytest.fixture()
async def supports_transactions(client: Client, table: TableName):
    try:
        await client.transact_write_items(
            [ConditionCheck(table, {"h": "h", "r": "r"}, F("h").does_not_exist())]
        )
    except UnknownOperation:
        raise pytest.skip(f"Transactions not supported")
```

Then you can `@pytest.mark.usefixtures('supports_transactions')` the tests to skip on implementations that don't support it.

The benefit of this is also that if dynalite does eventually add support, we don't need to change anything, or if we add yet another implementation to our test suite we don't have to worry about it either.